### PR TITLE
Fix wrapper of pygenometracks for multiple bedgraphs and add test

### PR DESCRIPTION
**Pull request by lldelisle**
FOR CONTRIBUTOR:
* [X] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [X] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [X] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Review by bgruening — APPROVED** [review]

**Comment by lldelisle** [comment]
Ready to be merged :wink:

File: tools/pygenometracks/pyGenomeTracks.xml [hunk 1]
```diff
@@ -426,7 +426,7 @@ show_data_range = $track.track_file_style_conditional.show_data
 grid = $track.track_file_style_conditional.grid
 nans_to_zeros = $track.track_file_style_conditional.nans_to_zeros
 use_middle = $track.track_file_style_conditional.use_middle
-      #set $columns = len(open(str($track.track_file_style_conditional.track_input_bedgraph)).readline().split('\t'))
+      #set $columns = len(open(str(data_bedgraph)).readline().split('\t'))
```
Comment (bernt-matthias) [inline tools/pygenometracks/pyGenomeTracks.xml:429]: Wondering if you can't get this from the metadata, i.e. `data_bedgraph.metadata.columns`?

Comment (lldelisle) [reply tools/pygenometracks/pyGenomeTracks.xml]: You are right. This is much prettier.

**Comment by lldelisle** [comment]
Thanks

**Pull request merged**